packages/builder/src/components/portal/licensing/AccountLockedModal.svelte

```diff
@@ -0,0 +1,31 @@
+<script>
+  import { Modal, ModalContent, Body } from "@budibase/bbui"
+
+  let modal
+
+  export let onConfirm
+
+  export function show() {
+    modal.show()
+  }
+
+  export function hide() {
+    modal.hide()
+  }
+</script>
+
+<Modal bind:this={modal} on:hide={modal}>
+  <ModalContent
+    title="Your account is currently locked"
```

Comment:
Think it's a bit confusing how we mix in two terminologies here
- account is locked
- account has been de-activated

Might be worth asking andy on a unified terminology to use here